src/doc/src/reference/resolver.md

```diff
@@ -517,6 +551,17 @@ But, if you run into this situation, the [`cargo update`] command with the
 
 [`cargo update`]: ../commands/cargo-update.md
 
+### Why wasn't a newer version selected?
```

Comment:
nit: we can link to an issue having some guidance for debugging as an example. I can't recall any specific one at this moment but I believe there are some.

Reply:
If you have one, that'd be great.  I can't think of ay.